calendarpal/src/main/java/org/dmfs/android/calendarpal/tables/CalendarScoped.java

```diff
@@ -102,7 +100,7 @@ public CalendarScoped(@NonNull RowSnapshot<CalendarContract.Calendars> calendarR
 
     private Predicate calendarScoped(@NonNull Predicate predicate)
     {
-        return new AllOf(predicate, new EqArg(CalendarContract.Events.CALENDAR_ID,
-                mCalendarRow.values().charData(CalendarContract.Calendars._ID).value("-1")));
+        return new org.dmfs.android.calendarpal.predicates.CalendarScoped(mCalendarRow, predicate);
+
```

Comment:
please watch out for empty lines like this one.

Reply:
done